src/mongoose_redis.erl

```diff
@@ -7,6 +7,7 @@
 -module(mongoose_redis).
 -author("bartlomiej.gorny@erlang-solutions.com").
 -include("mongoose.hrl").
+-include_lib("eredis/include/eredis.hrl").
```

Comment:
is `return_value()`  defined in `eredis.hrl`,  it seems to be a wrong approach

Reply:
say it to eredis authors